packaging/DISTRIBUTIONS.md

```diff
@@ -0,0 +1,38 @@
+# Netdata distribution support matrix
+![](https://raw.githubusercontent.com/netdata/netdata/master/web/gui/images/packaging-beta-tag.svg?sanitize=true)
+
+In the following table we provide the list of officially supported operating systems.
```

Comment:
Maybe replace this paragraph with this: `In the following table we've listed Netdata's official supported operating systems. We detail the distributions, flavors, and the level of support Netdata is currently capable to provide.`